pages/docs/tutorials/getting-started/request_reply.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Request and reply pattern
+weight: 20
+---
+
+A quite common messaging pattern is [request-reply](https://www.enterpriseintegrationpatterns.com/patterns/messaging/RequestReply.html), which describes a **requester**, which sends a request message and waits for a reply and a **replier** which receives the request and responds with a reply.
+
+As of AsyncPAI v3, this pattern can be described natively.
```

Comment:
```suggestion
```

1. AsyncPAI? 😂
2. We don't mention different spec versions; we must write in the PRESENT tense. _(i.e., only the present spec version is relevant and thus needs no mention)_

Reply:
Makes sense 👍 

Wups 😆